.github/workflows/linting.yml

```diff
@@ -40,3 +40,14 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Check DAOS logging macro use.
         run: ./utils/cq/d_logging_check.py --github src
+
+  ftest-tags:
+    name: Ftest tag check
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Normal behaviour is to checkout a merge commit with the target branch, this code will checkout the branch itself.

The difference comes in annotating against line numbers, with a merge commit they might be wrong although of course with a branch checkout you're not testing what would be landed.  Unless you're annotating the PR against specific lines in files you probably don't want this, if you're just failing the build with logging or annotating against files (which I think is done against "line 0" then you don't want to specify a ref here.

Reply:
Thanks. The ftest lint just fails with logging + exit(1). No specific lines or files are annotated. So I think I want the default behavior